index.js

```diff
@@ -1,6 +1,13 @@
 var postcss = require('postcss')
+var postcssrc = require('postcss-load-config')
 
 module.exports = function (plugins, options, filterType) {
+  if (Array.from(arguments).length === 0) {
```

Comment:
the `arguments` has the property `length` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments/length

Reply:
Thanks, fixed.